executor/runtime/docker/docker.go

```diff
@@ -574,6 +575,10 @@ func (r *DockerRuntime) dockerConfig(c *runtimeTypes.Container, binds []string,
 		c.Env["EC2_OWNER_ID"] = vpcAccountID
 	}
 
+	if requireToken, ok := c.TitusInfo.GetPassthroughAttributes()[imdsRequireToken]; ok {
```

Comment:
Maybe we should rename REQUIRE_TOKEN to something else. We typically "control" variables under the TITUS_* namespace. So, maybe we should call it like TITUS_IMDS_REQUIRE_TOKEN?

(It'd require a change in two places)

Reply:
Sure thing, done. Let me know what you think.